response/response.go

```diff
@@ -21,8 +21,8 @@ const (
 	StatusFail = "fail"
 )
 
-// ResponseInterface - Interface for microservice responses.
-type ResponseInterface interface {
+// Interface - Interface for microservice responses.
+type Interface interface {
```

Comment:
I know that 'er names are usually reserved for single-method interfaces but I think `Responder` would be better here than `Interface`...

Reply:
`sort.Interface` from the `sort` package was my inspiration here. Though I am happy to change this!

Reply:
changed